### DISCUSSION TITLE
how do you convert c source to v?

### DISCUSSION THREAD

**Discussion by MrMidnightOfficial** [vlang/v#14820] [General]
how do you convert c source to v?

when I do `v translate` it gives me a output:
C2V is not installed. Cloning C2V to C:\Users\Dragon boy\.vmodules\c2v_alpha ...
Failed to download C2V. Perhaps it is not released yet? Is it June 20 yet?

**Comment by larpon** [comment]
By hand, until it can be done automatically

**Comment by medvednikov** [comment]
C2V has just been released:

https://twitter.com/v_language/status/1539689311636955136
